client/packages/system/src/Manage/Preferences/EditPage/PreferenceLabelRow.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactNode } from 'react';
+import { InputWithLabelRow } from '@openmsupply-client/common';
+
+interface PreferenceLabelRowProps {
+  label: string;
+  Input: ReactNode;
+  isLast?: boolean;
+}
+
+export const PreferenceLabelRow = ({
+  label,
+  Input,
+  isLast = false,
+}: PreferenceLabelRowProps) => {
+  return (
+    <InputWithLabelRow
+      labelWidth={'100%'}
+      label={label}
+      Input={Input}
+      sx={{
```

Comment:
This just keeps styling consistent.
I've now made both store and global prefs show the label on the left...